docs/docs/userguide/CONFIGURATION.md

```diff
@@ -5,7 +5,8 @@ In order to provision the infrastructure and install an Atlassian Data Center pr
 The content of the configuration file is divided into two groups:
 
 1. [Mandatory configuration](#mandatory-configuration)
```

Comment:
I think we should change this category fundementally. This was designed when we had just one product. Now we have three products and the new category should be :

1. Common configuration
2. Product specific Configuration

   a. Jira Configuration

   b. Confluence Configuration

   c. Bitbucket Configuration

   d. Bamboo Configuration
   

Obviousely there are some optional and mandetory settings in each group and those could be define by adding `(optional)` when we introduce the setting. See `config.tfvars`.

Reply:
Are you saying `Mandatory configuration` should be renamed to `Common configuration`? Additionally isnt the current optional config universal to all products hence it being in a section of its own?

Reply:
As we have multi-product, the mandatory and optional concept is not making sense now. We have _common configuration_ and _product specific configuration_. In each category some items are mandatory (have no default value) and some are optional (have default value). Refactoring the documentation in this section is needed in my opinion.

Reply:
I've updated things again - best viewed via `make docs`. Let me know what you think.